operator-common/src/main/java/io/strimzi/operator/common/Annotations.java

```diff
@@ -79,6 +79,11 @@ public class Annotations {
      */
     public static final String ANNO_STRIMZI_IO_PAUSE_RECONCILIATION = STRIMZI_DOMAIN + "pause-reconciliation";
 
+    /**
+     * Annotation used to bypass the broker scale-down mechanism
+     */
+    public static final String ANNO_STRIMZI_IO_SKIP_BROKER_SCALEDOWN_CHECK = STRIMZI_DOMAIN + "skip-broker-scaledown-check";
+
```

Comment:
Note for the future: Depending on what is merged first, this should reconcile with https://github.com/strimzi/strimzi-kafka-operator/pull/9103 as this is a public annotation. (nothing to do right now, we just need to keep it in mind)